protocol/persistence.go

```diff
@@ -1134,6 +1134,58 @@ func (db sqlitePersistence) StatusUpdates() (statusUpdates []UserStatus, err err
 	return
 }
 
+func (db sqlitePersistence) DeleteSwitcherCard(id string) error {
+	_, err := db.db.Exec("DELETE from switcher_cards WHERE id = ?", id)
+	return err
+}
+
+func (db sqlitePersistence) AddSwitcherCard(switcherCard SwitcherCard) error {
```

Comment:
Because of `ON CONFLICT REPLACE`, I would call this endpoint `Upsert` (and the APIs etc), as it better reflects what it does (`insert or update` essentially)